### DISCUSSION TITLE
Max_radius parameter in `trajectory_aggregator`

### DISCUSSION THREAD

**Discussion by krober10nd** [anitagraser/movingpandas#232] [Q&A] [answered]
Per the original paper (http://geoanalytics.net/and/papers/tvcg11.pdf) for generalizing trajectories, the term `max_radius` which is used in the clustering of characteristic points is not necessarily equal to the parameter `maximum_distance`. However, in the current implementation these two parameters are equal.

Shouldn't these two parameters be specified separately? Could a new kwarg could be added to `TrajectoryCollectionAggregator` called `max_radius`?

**Comment by anitagraser** [comment, marked as answer]
I don't remember a specific reason for equating max_radius and maximum_distance. It's probably an oversight on my part. Feel free to add the necessary kwarg and provide a PR.
